package.json

```diff
@@ -75,6 +73,8 @@
     "react": ">=0.12.0"
   },
   "dependencies": {
+    "babel-cli": "^6.24.1",
```

Comment:
Why were these moved here? They shouldn't be in dependencies since we build the application to not require any dependencies other than React